### DISCUSSION TITLE
EL7 and Ubuntu-18

### DISCUSSION THREAD

**Discussion by daiyam** [VSCodium/vscodium#1969] [General]
Hi All,

Since `1.91`, VSCodium has stopped to work on EL7 and Ubuntu-18.
But the REH(server) releases are still working on those systems.

What should we do? Is there a need for the Client to be working on the those systems?

Can you comment if you care?

**Comment by sskamesh** [comment]
I still use EL7 most of the time (especially at work). But then since EL7 went EOL on June 2024, not sure if it's viable to maintain it for EL7 anymore though. I'd vote yes, if it's not too much work.
